docs/tutorials/dronecan_firmware_upgrade.rst

```diff
@@ -0,0 +1,134 @@
+an ongoing file transfer:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/firmware_updating.png
+
+After the upgrade is complete, your module will reboot and resume normal operation.
+
+Viewing Firmware Upgrade Status with the Bus Monitor
+---------------------------------------------------------
+If you would like to view the details of the firmware upgrade process, you can use the DroneCAN GUI's *Bus Monitor* tool. You can find the Bus Monitor under Tools > Bus Monitor
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/tools_bus_monitor.png
+
+The Bus Monitor tool allows you to view all DroneCAN messages being sent through the DroneCAN bus. For example, this is where you can view all of your connected 
+modules' ESC statuses. In order to start monitoring the DroneCAN bus, click the camera icon in the top left:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/monitor_camera_button.png
+
+With modules connected and powered on, you will see ``uavcan.equipment.esc.Status`` messages appear. Click on a message to view the decoded data in the bottom left.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/decoded_esc_status.png
+
+In order to inspect firmware update status with the Bus Monitor tool, start capturing bus activity, then trigger a firmware upgrade. You will see the USB-to-CAN node 
+transmit the ``uavcan.protocol.file.BeginFirmwareUpdate`` message, a handshake between the two nodes about the file's information through 
+the ``uavcan.protocol.file.GetInfo`` message, and then finally, the transfer of file information from the USB-to-CAN node to the Vertiq node through the 
+``uavcan.protocol.file.Read`` message.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/monitor_read_file.png
+
+Firmware Update While Connected with a Flight Controller
+=============================================================
+An important feature of CAN is that it is a fully bussable protocol. As such, multiple DroneCAN devices can be connected to the same bus simultaneously without issue. 
+Suppose you have configured your DroneCAN hardware as recommended by `PX4's documentation <https://docs.px4.io/main/en/can/#wiring>`_.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/px4_wiring_pic.png
+    :scale: 70%
+
+Now, you would like to update your Vertiq module firmware without dismantling your vehicle. Simply add your USB-to-CAN device to the bus chain, and connect it to the DroneCAN GUI tool. 
+In this example, we are connected to only one Vertiq module, do not have a GNSS device, and are using only one CAN output. So, our chain is as follows:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_upgrade_bus.png
+
+Take care to ensure that you connect all CANH signals together and all CANL signals together. If any connection is backwards, you will not be able to communicate with 
```

Comment:
I think a lot of this information is just general CAN bus stuff, not really anything specific to this example. You maybe can cut it down to just generally saying that they should connect to the bus with the proper connections and termination, they can use the second set of CAN connections if they'd like, and go check out your family page for more details. The CANH/CANL stuff isn't something we need to specify here, its just how CAN works.